tools/aws_lab_setup/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -22,3 +29,181 @@ ec2_lab_node_types:
 
 #  - name: haproxy
 #    type: rhel7
+
+ec2_lab_node_types_networking:
+  - name: ansible
+    type: rhel7-tower
+
+  - name: rtr1
+    type: csr-byol
+
+  - name: rtr2
+    type: csr-byol
+
+  - name: host1
+    type: rhel7
+
+# --- Lookup tables --- #
+# Region specific AMIs, subnets, and VPCs
+
+ec2_regions:
+  us-east-1:
+    vpc: vpc-06c9a463
```

Comment:
Any problem having these vpc IDs in a public repo?

Reply:
Yes I do. I guess #116 should have stated no hard-coded VPCs of any sort more explicitly.

Reply:
@IPvSean any way to strip these out and provide them to folks running the lab separately? It does seem like they ought to be parameterized.

Reply:
@gregdek @tima this was not actually something I did.  What I did was move the variables to a common location (from tools/aws_lab_setup/roles/manage_ec2_instances/vars to tools/aws_lab_setup/roles/manage_ec2_instances/defaults).  Prior to that there was vars in both locations.  There was a public slack conversation about if this was OK and I got approval to move forward.  This is showing up as green just for the vars that moved locations, but I didn't change anything here.  Personally I think we should change to the [ec2_ami_find](http://docs.ansible.com/ansible/latest/ec2_ami_find_module.html) but I think this should come in as a separate PR.  I don't mind doing that but right now this is how lightbulb was working.

Reply:
I don't know the best way to fix this, but hard-coding VPC IDs seems to be a bad idea ¯\_(ツ)_/¯

And it's true, this hardcoding already exists in my local copy of upstream, in tools/aws_lab_setup/roles/manage_ec2_instances/vars/main.yml.

So given that, maybe this is a separate issue with a separate PR.

Reply:
I realize you didn't do it @IPvSean and I'm not assigning blame -- just asking it get addressed. I don't think removing some default values and adding it to the docs to big an ask. I too prefer the use of `ami_find` and made that a requirement of the overhaul. I would have rather seen that done, but I'm OK accepting some triage measures.

Reply:
@tima is there an AI for me on this issue to get this PR accepted?

Reply:
I don't mind ripping out all the VPC hardcoded all together I just have no idea who that will affect or what it will break... it does not affect the networking stuff at all, all of that is pre-existing